raiden/settings.py

```diff
@@ -59,3 +68,20 @@
 MONITORING_MIN_CAPACITY = TokenAmount(100)
 
 MEDIATION_FEE = FeeAmount(0)
+
+
+MEDIATION_FEE_CONFIG_KEY = "mediation_fees"
```

Comment:
Having the dict key as a constant feels like a weird middle ground between
* the simple way of having just x['key'] (which is what we do in most places when accessing the config)
* and the type safe way of having a data class and using x.key

Changing the config to a dataclass is probably a sizable PR just by itself.

Do we use the constant-as-key approach regularly? Otherwise we should probably not introduce it to keep the code consistent.

Reply:
Yeah, changing the config to be a dataclass will take some effort.

I changes to the dict key constant because it was horribel to change all these settings back, but I see your point about consistency. I'll change it.